contexts/credentials/v2

```diff
@@ -0,0 +1,216 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+
+    "id": "@id",
+    "type": "@type",
+
+    "VerifiableCredential": {
+      "@id": "https://www.w3.org/2018/credentials#VerifiableCredential",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+
+        "id": "@id",
+        "type": "@type",
+
+        "name": {"@id": "https://www.w3.org/2018/credentials#name", "@type": "@id"},
+        "description": {"id": "https://www.w3.org/2018/credentials#description", "@type": "@id"},
```

Comment:
```suggestion
        "name": "http://schema.org/name",
        "description": "http://schema.org/description",
```

Reply:
You might as well add `image` while you're in here.

Reply:
Thanks was unclear if we wanted to define ourselves or use schema.org